[2025-09-17T06:34:26.651+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: opensky_data_pipeline_dag.dbt_run_silver manual__2025-09-17T06:34:07.403725+00:00 [queued]>
[2025-09-17T06:34:26.653+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: opensky_data_pipeline_dag.dbt_run_silver manual__2025-09-17T06:34:07.403725+00:00 [queued]>
[2025-09-17T06:34:26.653+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-17T06:34:26.657+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): dbt_run_silver> on 2025-09-17 06:34:07.403725+00:00
[2025-09-17T06:34:26.659+0000] {standard_task_runner.py:60} INFO - Started process 8839 to run task
[2025-09-17T06:34:26.660+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'opensky_data_pipeline_dag', 'dbt_run_silver', 'manual__2025-09-17T06:34:07.403725+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/opensky_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpdrk15wd7']
[2025-09-17T06:34:26.661+0000] {standard_task_runner.py:88} INFO - Job 56: Subtask dbt_run_silver
[2025-09-17T06:34:26.678+0000] {task_command.py:423} INFO - Running <TaskInstance: opensky_data_pipeline_dag.dbt_run_silver manual__2025-09-17T06:34:07.403725+00:00 [running]> on host 769d9d26935d
[2025-09-17T06:34:26.775+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='opensky_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='dbt_run_silver' AIRFLOW_CTX_EXECUTION_DATE='2025-09-17T06:34:07.403725+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-17T06:34:07.403725+00:00'
[2025-09-17T06:34:26.775+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-17T06:34:26.775+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/airflow/flightweather_dbt && dbt run --select tag:silver --profiles-dir .']
[2025-09-17T06:34:26.779+0000] {subprocess.py:86} INFO - Output:
[2025-09-17T06:34:27.288+0000] {subprocess.py:93} INFO - [0m06:34:27  Running with dbt=1.10.11
[2025-09-17T06:34:27.366+0000] {subprocess.py:93} INFO - [0m06:34:27  Registered adapter: postgres=1.9.1
[2025-09-17T06:34:27.484+0000] {subprocess.py:93} INFO - [0m06:34:27  Encountered an error:
[2025-09-17T06:34:27.485+0000] {subprocess.py:93} INFO - Compilation Error in model silver_secondary_table (models/opensky/silver_secondary_table.sql)
[2025-09-17T06:34:27.485+0000] {subprocess.py:93} INFO -   expected token ',', got 'info'
[2025-09-17T06:34:27.485+0000] {subprocess.py:93} INFO -     line 27
[2025-09-17T06:34:27.485+0000] {subprocess.py:93} INFO -       {% do log("silver_main_table at " ~ timestamp ~ " | Total rows: " ~ timestamp info=True) %}
[2025-09-17T06:34:28.584+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-09-17T06:34:28.590+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-09-17T06:34:28.591+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=opensky_data_pipeline_dag, task_id=dbt_run_silver, execution_date=20250917T063407, start_date=20250917T063426, end_date=20250917T063428
[2025-09-17T06:34:28.595+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 56 for task dbt_run_silver (Bash command failed. The command returned a non-zero exit code 2.; 8839)
[2025-09-17T06:34:28.642+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-09-17T06:34:28.652+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
